paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CloneActionITCase.java

```diff
@@ -504,6 +766,7 @@ private FileStoreTable getFileStoreTable(String warehouse, String db, String tab
     //  Random Tests
     // ------------------------------------------------------------------------
 
+    @Disabled
```

Comment:
Why disable this test?

Reply:
Because current clone  design, we can not deal with expiration snapshots, as we have discussed offline, we do not deal with it now.